packages/shared/package.json

```diff
@@ -65,6 +65,7 @@
         "npm": ">=6.14.11"
     },
     "devDependencies": {
+        "@nangohq/records": "file:../records",
```

Comment:
I couldn't entirely avoid making `records` a dependency of `shared` because some `run.service` integration end-to-end tests are accessing the db but at least it is only a dev dependency